crates/mun_runtime/src/lib.rs

```diff
@@ -155,27 +156,37 @@ impl MunRuntime {
     }
 }
 
+/// Extends a result object with functions that allow retrying of an action.
+pub trait Retriable: Sized {
```

Comment:
Are we happy with this trait name? Or would something like `RetryResultExt` or `RetryExt` be better? `Ext` meaning extension, as this is a trait extension.

Reply:
RetryResultExt sounds good